src/services/CalculatorProcess.js

```diff
@@ -1,51 +1,50 @@
-let memory = [];
-let result = "";
-function calculatorProcess(content,visor) {
-	
-	const numbersOperators = () => String(eval((result)));
-	
-	memory += content;
+let memory = "";
+let sLength;
 
-	if (memory.includes("=")) {
+function CalculatorProcess(content, visor) {
 
-		result = memory.slice(0,-1);
+	const firstInteraction = ["%", "/", "*", "+", "-", "S", "C", "="];
```

Comment:
It's not clear 

What first Interaction means

Reply:
it's for when the first buttons clicked are one of those content,they should return nothiing